src/game/logic/Board.java

```diff
@@ -52,10 +52,19 @@ public Board(){
      * @param index
      * @return Cell
      */
-    public Cell getCell(int index){
+    public Cell getCell(int index){ // i wrote this
```

Comment:
Nit Pick: Git/Github will tell us what parts you changed so there's no need to put the comment here

Reply:
it's a note for myself just so I knew what I wrote